cogment_verse/constants.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-COGMENT_VERSION = "v2.5.0"
+COGMENT_VERSION = "v2.8.0"
```

Comment:
I think 2.9 is out :)

Reply:
For some reasons, 2.9.0 has not worked yet :)